jobs/shareMethod.groovy

```diff
@@ -41,7 +41,9 @@ def getLockedResourceName(resources,label_name){
 
 def buildAndPublish(){
     stage("Packages Build"){
-        load("jobs/build_debian/build_debian.groovy")
+        retry(3){
```

Comment:
Thanks for your review!
The item 1 is fixed.
About item2:
3 is just a experience value